compare.md

```diff
@@ -1,79 +1,105 @@
-
 Document ID: 12J_waJEbC1EEBL3MVy_RqyOWfh6ZslV45b-csN4B7Bs_0
```

Comment:
Left one is: Davinci _____________________________________________________________________  Right one is: Turbo